src/test/java/com/wikia/webdriver/testcases/mercurytests/SmartBannerTests.java

```diff
@@ -52,43 +50,27 @@ private Colors(String hex) {
       {"lego", "LEGO_Wiki", Colors.YELLOW.hex},
       {"ladygaga", "Gagapedia", Colors.MAGENTA.hex}
   };
-  private static final String BUTTON_NAME_FOR_ANDROID = "Install";
-  private static final String BUTTON_NAME_FOR_IOS = "GET";
 
   @Test(groups = "MercurySmartBannerTest_001")
-  @RelatedIssue(issueID = "XW-656")
-  @InBrowser(browser = Browser.CHROME_ANDROID)
+  @InBrowser(browser = Browser.CHROME_MOBILE_MERCURY)
   public void MercurySmartBannerTest_001_ButtonName_FixPosition_Close() {
     SmartBannerComponentObject banner = new SmartBannerComponentObject(driver);
     wikiURL = urlBuilder.getUrlForWiki(WIKIS[0][0]);
     banner.openArticleOnWikiByNameWithCbAndNoAds(wikiURL, WIKIS[0][1]);
-    PerformTouchAction touchAction = new PerformTouchAction(driver);
-
-    String buttonName;
+    MobileEmulationTouchActions touchActions = new MobileEmulationTouchActions(driver);
 
     Assertion.assertTrue(
         banner.isSmartBannerVisible(),
         "Smart banner is closed"
     );
 
-    if ("ANDROID".equals(Configuration.getPlatform())) {
-      buttonName = BUTTON_NAME_FOR_ANDROID;
-    } else {
-      buttonName = BUTTON_NAME_FOR_IOS;
-    }
+    Assertion.assertEquals(banner.isButtonVisible() == true, true, "Button is not visible");
```

Comment:
you can use 
`Assertion.assertTrue(banner.isButtonVisible(), "Button is not visible");`